test/signer_test.exs

```diff
@@ -4,10 +4,13 @@ defmodule AntikytheraAws.Signer.V4Test do
   use Croma.TestCase
   alias AntikytheraAws.Auth.Credentials, as: Creds
 
-  @example_id     "AKIDEXAMPLE"
-  @example_secret "wJalrXUtnFEMI/K7MDENG+bPxRfiCYEXAMPLEKEY"
-  @example_creds  %Creds{access_key_id: @example_id, secret_access_key: @example_secret}
-  @example_region "us-east-1"
+  @example_id      "AKIDEXAMPLE"
+  @example_secret  "wJalrXUtnFEMI/K7MDENG+bPxRfiCYEXAMPLEKEY"
+  @example_creds   %Creds{access_key_id: @example_id, secret_access_key: @example_secret}
+  @example_region  "us-east-1"
+  @example_payload "example payload"
+
+  defp hex_sha256(str), do: Base.encode16(:crypto.hash(:sha256, str), case: :lower)
```

Comment:
Isn't it possible to reuse `V4.hex_sha256/1` by defining it as `defpt`?

Reply:
Yes, fixed in 2fed775
(I only knew `defunpt`...)